### ISSUE TITLE
Hide floating buttons/icons on display

### ISSUE THREAD

**Issue by CrimsonBloodfang** [Nain57/Smart-AutoClicker#779]
### Describe the feature

We need a toggle to hide and unhide the floating buttons without violating Android accessibility guidelines of course. Near-invinsible or side-collapsed to a few pixels (the minimum interactive size) or transfer them to notification? Hide and unhide by tapping an icon or maybe through gesture or volume buttons (combination)? Preferably not bound by Google Play's accessibility policy to achieve this.

### Describe the usage

This is a very big improvement to the reading experience for everyone who uses this as a simple ‘auto-scroll’ app, keeping the controls hidden so the display stays unobstructed.

### Is it a duplicate?

- [x] There is no duplicates
